config/environments/production.rb

```diff
@@ -3,8 +3,6 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
-  config.x.read_only_database_url = ENV.fetch('DATABASE_URL', nil)
```

Comment:
Do we want to add this line back in? I don't know what it does or why it's needed 🤷‍♂️

Reply:
I think this was a work around from a long time ago but happy to put it back in and we can remove it in a more targeted PR. The only reason I removed it here is because the auto-generator did it.

https://github.com/DFE-Digital/publish-teacher-training/pull/4445/commits/4c12fceeeff54904dad3ac4c75050f527c9c0399